plugins/star-rating/frontend/public/templates/feedback-popup.html

```diff
@@ -162,6 +162,8 @@
                 var device_id = getParameterByName('device_id');
                 var sdk_version = getParameterByName('sdk_version');
```

Comment:
Can we have fallback for SDK version too, because here from debug perspective I am interested more in the feedback plugin version, rather than SDK version.